### DISCUSSION TITLE
Could you please advise on how to read a file in a serverless function?

### DISCUSSION THREAD

**Discussion by spb-web** [vercel/community#3793] [Help] [answered]
framework: nuxt 3
nitro preset: vercel

```
const resolvePath = (value: string) => path.join(process.cwd(), value)
const result = await fs.readdir(resolvePath('./footballCaptchaAssets/objects'))
```

Error:
```
[nuxt] [request error] [unhandled] [500] ENOENT: no such file or directory, scandir '/var/task/footballCaptchaAssets/objects'
```

**Comment by spb-web** [comment]
similar problem: #669

**Comment by amyegan** [comment, marked as answer]
The advice that usually works is in this guide: https://vercel.com/guides/how-can-i-use-files-in-serverless-functions

It looks slightly different from the code snippet you shared. Please let me know if that works for you 🙏

**Reply by spb-web** [reply]
Thank you!)
I added vercel.json with a build command override, and after that, the issue was resolved.

```
{
  "buildCommand": "nuxt build && cp -R ./footballCaptchaAssets .vercel/output/functions/__nitro.func/footballCaptchaAssets"
}
```
